oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -145,3 +153,53 @@ def _populate_create_date(self):
         for date in dates:
             self.create_origin_info()
             self.origin_info.created.append(mods.DateCreated(date=date))
+
+    def _populate_constituent_audio(self):
+        # For each child of the item, get submaster audio MediaFile
+        for child in ProjectItem.objects.filter(parent=self._item).order_by("sequence"):
+            for audiofile in MediaFile.objects.filter(
+                Q(item=child) & Q(file_type__file_code="audio_submaster")
+            ):
+                self.related_items.append(self._create_relateditem_audio(audiofile))
+
+    def _create_relateditem_audio(self, mi: MediaFile) -> MODSv34:
+        pi = mi.item
+        ri = RelatedItemOH(href=mi.file_url, type="constituent")
+        ri.title = pi.title
+        ri.identifiers.append(mods.Identifier(text=pi.ark))
+        ri.parts.append(PartOH(order=pi.sequence, type="session_audio"))
+
+        for toc in Description.objects.filter(item=pi, type__type="tableOfContents"):
+            ri.toc = TableOfContents(text=toc.value)
+
+        for ts in MediaFile.objects.filter(
+            Q(item=pi) & Q(file_type__file_code="text_master_index")
+        ):
+            if ts.file_url != "":
+                ri.locations.append(
+                    LocationOH(url=ts.file_url, usage="primary display")
+                )
+
+        return ri
+
+
+# Extended classes to supply some additional attributes not in stock library that we use
+
+
+class LocationOH(mods.Location):
+    usage = xmlmap.StringField("mods:url/@usage")
+
+
+class PartOH(mods.Part):
+    order = xmlmap.StringField("@order")
+
+
+class TableOfContents(Common):
+    ROOT_NAME = "tableOfContents"
+    text = xmlmap.StringField("text()")
+
+
+class RelatedItemOH(mods.RelatedItem):
+    MODS.ROOT_NAMESPACES["xlink"] = "http://www.w3.org/1999/xlink"
```

Comment:
Should this be `MODSv34.ROOT_NAMESPACES` for consistency with the other `MODSv34` uses?  This is the only place `MODS` is referenced, and tests still pass when changed to `MODSv34`.